decred/tests/unit/dcr/test_txscript.py

```diff
@@ -812,10 +812,14 @@ def ssgenMsgTxWrongFirstOut():
 
 
 class TestSubsidyCache:
+    """
+    Based on dcrd's TestSubsidyCacheCalcs.
```

Comment:
This isn't the basis for the entire class, but just the `test_subsidy_cache_calcs` function. Same goes for the other removed names.